hypertrace-alert-engine/build.gradle.kts

```diff
@@ -29,10 +29,10 @@ dependencies {
 
     // Logging
     implementation("org.slf4j:slf4j-api:1.7.30")
-    runtimeOnly("org.apache.logging.log4j:log4j-slf4j-impl:2.14.1")
+    runtimeOnly("org.apache.logging.log4j:log4j-slf4j-impl:2.15.0")
 
     // framework + libs
-    implementation("com.google.protobuf:protobuf-java-util:4.0.0-rc-2")
+    implementation("com.google.protobuf:protobuf-java-util:3.17.3")
```

Comment:
This one's actually a downgrade. Assuming all the tests are passing I'm going to take that as OK. We shouldn't be using prerelease artifacts, and we should be aligning with the version of protoc we're using elsewhere.

Reply:
@kotharironak was there any specific reason behind using this version?

Reply:
No. I don't think, there was any specific reason. I might have chosen directly from maven with latest one - https://mvnrepository.com/artifact/com.google.protobuf/protobuf-java

This is fine.